src/web-ui/src/public/Footer.vue

```diff
@@ -12,9 +25,38 @@
 </template>
 
 <script>
+import { RepositoryFactory } from '@/repositories/RepositoryFactory'
+//import { AnalyticsHandler } from '@/analytics/AnalyticsHandler'
+import AmplifyStore from '@/store/store'
+
+const UsersRepository = RepositoryFactory.get('users')
+
 export default {
   name: 'Footer',
   props: {
+  },
+  computed: {
+    user() { 
+      return AmplifyStore.state.user
+    }
+  },
+   created () {
+    this.getUser(AmplifyStore.state.user.id)
+  },
+  methods: {
+    async getUser(userID) {
+      if (userID && userID.length > 0) {
+        const { data } = await UsersRepository.getUserByID(userID)
+        this.user = data
+      }
+      return this.user
+    },
+    async verifyAndUpdateUserPhoneNumber() {
+      var userPhoneNumber = document.getElementById('sms-alerts').value
+      const { data } = await UsersRepository.verifyAndUpdateUserPhoneNumber(AmplifyStore.state.user.id, userPhoneNumber)
+      console.log(data)
```

Comment:
Should we have some better error handling here? Like displaying a message when a bad phone number is entered and the API returns a response telling us so.

Reply:
Good point. I can add it.